wpsc-components/merchant-core-v3/gateways/ec-incontext.js

```diff
@@ -0,0 +1,12 @@
+/*
+* Incontext Checkout for Express Checkout
+*/
+window.paypalCheckoutReady = function () {
+	paypal.checkout.setup( wpec_ppic.mid, {
```

Comment:
Let's declare the `paypal` and `wpec_ppic` variables as global at the top of this file.  [Like this](https://github.com/wp-e-commerce/WP-e-Commerce/blob/master/wpsc-admin/js/admin.js#L1).